### PR TITLE
Update gallery.yaml with codex-clustergrammer2

### PR DESCRIPTION
**Pull request by cornhundred**
My example runs on MyBinder (see https://github.com/ismms-himc/codex_dashboard) and I followed the https://github.com/voila-gallery/gaussian-density example. The example shows how we can use our custom widget https://github.com/ismms-himc/clustergrammer2 and bqplot to visualize high-dimensional single cell data. Here's a tutorial link http://www.youtube.com/watch?v=JlUvt4rpF-s

### CODE REVIEW

**Comment by jtpio** [comment]
Thanks @cornhundred, that's a cool example!

File: tljh-voila-gallery/tljh_voila_gallery/gallery.yaml [hunk 1]
```diff
@@ -35,3 +35,9 @@ examples:
     url: voila/render/Interactive-Football-Pitch.ipynb
     repo_url: https://github.com/seidlr/voila-interactive-football-pitch
     image_url: https://i.imgur.com/yuWQZtL.png
+  codex-clustergrammer2:
+    title: codex-clustergrammer2
+    description: CODEX Single Cell Multiplexed Imaging Dashboard
+    url: voila/render/index.ipynb
+    repo_url: https://github.com/ismms-himc/codex_dashboard
+    image_url: https://github.com/ismms-himc/codex_dashboard/blob/master/img/codex_screenshot.png
```
Comment (jtpio) [inline tljh-voila-gallery/tljh_voila_gallery/gallery.yaml]: I think we'll need to use the "raw" url here:

```suggestion
    image_url: https://raw.githubusercontent.com/ismms-himc/codex_dashboard/master/img/codex_screenshot.png
```

**Comment by cornhundred** [comment]
We are continuing to develop the dashboard so any feedback is welcome :)

File: tljh-voila-gallery/tljh_voila_gallery/gallery.yaml (hunk 1 above)
Comment (cornhundred) [reply tljh-voila-gallery/tljh_voila_gallery/gallery.yaml]: thanks for the improvement

**Comment by jtpio** [comment]
Sounds good!

In the meantime we are going to add support for "pinning" examples (https://github.com/voila-gallery/gallery/pull/59).
It means that you can iterate on yours on master, and then open a new PR here to update with the new version once it's ready.

**Comment by cornhundred** [comment]
@jtpio Great! Thanks for the merge

**Comment by cornhundred** [comment]
@jtpio I'm seeing a 500 Internal server error when I try to run the codex-clustergrammer2 example from https://voila-gallery.org/services/gallery/

I raised the issue here: https://github.com/voila-gallery/gallery/issues/63

**Pull request merged**